### ISSUE TITLE
Provide custom color with better contrast for the current focused element outline

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#4288]
## Summary

Currently Clarity provides the default browser color for the focused element outline. This however does not provide enough contrast in some browsers(Chrome) or is not perceivable enough in others (Firefox)

The color of the outline should be custom with better contrast : at least 3.0:1.

* What is the change? - Please provide custom outline color with better contrast
* Why should it go in Clarity? - Accessibility enhancement for low vision users
* Does this change impact existing behaviors? If so how? - Introduces new color for the focused element outline
* If this change introduces a new behavior, is this behavior accessible? - Yes

## Examples
![tabs-ff](https://user-images.githubusercontent.com/2686283/74736134-c4c4fd80-525a-11ea-9613-066ad8d5f7e0.png)

![close-btn](https://user-images.githubusercontent.com/2686283/74736114-bd055900-525a-11ea-816b-005d3a9a60aa.png)

## Workarounds

N/A

**Comment by mathisscott** [comment]
@colinreedmiller and @chlane need to weigh in on this.

Currently, we rely on the browser's default.

**Comment by chlane** [comment]
The browser default is the starting place but when there are cases where it has low contrast due to the color scheme, then the visual focus indication needs to meet a 3:1 contrast ratio.

**Comment by gnomeontherun** [comment]
Each browser has a different focus outline, which makes this ask very difficult. If we change the way focus appears in one place but not others, then we are inconsistent and confusing for users. In fact conditionally changing the way focus appears for some components seems to conflict with other a11y guidelines.

Then this becomes something we would have to solve globally the way this request is framed. Is it our responsibility to fix Chrome's color contrast (which is different from browsers to browser _and_ platform to platform)? This would become a large breaking change to force such a behavior, and I am quite worried this would become a major problem in applications.

**Comment by colinreedmiller** [comment]
It is important to remember that using the browsers default leaves the onus on the browser to meet a11y whereas customizing effectively takes responsibility for a11y compliance. I recommend not changing the browser focus indication unless in the case of a color scheme that makes overriding necessary. It is also important to remember that each browser will have its own method such that taking responsibility must account for each one.

**Comment by chlane** [comment]
I was hoping this would be the Clarity consumer's responsibility and that Clarity would simply not suppress visual focus indication with outline:0. 

Here is the current level access solution. 

*:focus, a:focus {
    outline: 2px solid #d71ef7;
    border: 1px solid #ffffff;
}

The purple outline may not be what designer's want but it doesn't clash with the blue colors some browsers use as a default and many common color schemes. I've also seen a solution using box shadow together with outline to provide both a black and a white visual treatment that was visible over any background.

**Comment by chlane** [comment]
According to this we can stick with the browser default and let the user customize the outline.

https://www.w3.org/WAI/WCAG21/Techniques/general/G165

See the text below.

Example 1

The default focus indicator on Microsoft Windows is a one-pixel, black dotted line around the focused element. On a page with a dark background, this can be very difficult to see. The creator of the page uses the default, and the user customizes it in Windows to make it a bright color.

**Comment by chlane** [comment]
We can stick with the browser default for minimum compliance and refer to  https://www.w3.org/WAI/WCAG21/Techniques/general/G165 if questioned about it.

**Comment by gnomeontherun** [comment]
I'm going to close this as working as expected, since users can override this setting on their end and we don't want to be changing global focus styling that browsers implement by default.
